aspnetcore/data/ef-rp/intro/samples/cu50/Pages/Instructors/Index.cshtml.cs

```diff
@@ -47,18 +47,14 @@ public async Task OnGetAsync(int? id, int? courseID)
             if (courseID != null)
             {
                 CourseID = courseID.Value;
-                var selectedCourse = InstructorData.Courses
-                    .Where(x => x.CourseID == courseID).Single();
-                await _context.Entry(selectedCourse)
-                              .Collection(x => x.Enrollments).LoadAsync();
-                foreach (Enrollment enrollment in selectedCourse.Enrollments)
-                {
-                  await _context.Entry(enrollment).Reference(x => x.Student).LoadAsync();
-                }
-                InstructorData.Enrollments = selectedCourse.Enrollments;
+                IEnumerable<Enrollment> Enrollments = await _context.Enrollments
+                    .Where(x => x.CourseID == CourseID)                    
+                    .Include(i=>i.Student)
+                    .ToListAsync();                 
+                InstructorData.Enrollments = Enrollments;
             }
             #endregion
```

Comment:
No need to change this, but we've switched to `// </snippet_1>`. I have a script that replaces them all if you want it.

Reply:
Do you mean the naming pattern has changed too (from descriptive name to "snippet" + sequence number) or we're just changing syntax from regions to XML tags?

Reply:
Naming pattern hasn't changed, we're just changing syntax from regions to XML tags